relational_database/homework.py

```diff
@@ -105,7 +128,14 @@ def task_8_count_customers_by_city(cur):
     Returns: 69 records in descending order
 
     """
-    pass
+    cur.execute("""
+        SELECT COUNT(CustomerName), city 
+        FROM customers 
+        GROUP BY city 
+        ORDER BY city DESC
+    """)
+    res = cur.fetchall()
+    return res
```

Comment:
```suggestion
    return cur.fetchall()
```
так виглядає акуратніше, + не резервує пам'ять для зайвої змінної

Reply:
відрефакторив)